src/api/context_config.c

```diff
@@ -234,6 +236,7 @@ static const bool fragment2iflag[NUM_ARITH_FRAGMENTS+1] = {
   true,   // NIA
   false,  // NRA
   true,   // NIRA
+  false,  // NFFA
```

Comment:
do we distinguish between linear and nonlinear FFA? 
if not, change the comment from NFFA -> FFA